test/test-applications/integrations/Samples.Http/TcpPortProvider.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="TcpPortProvider.cs" company="OpenTelemetry Authors">
```

Comment:
This file is duplicating test/integration-tests/IntegrationTests.Helpers/TcpPortProvider.cs. Is there any reason why not to add a reference to the project that contains it?

Reply:
I don't want for a sample to reference any other projects here. I just find it odd - it should be independent